src/platform/intel/cavs/include/cavs/lp_ctx.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Marcin Maka <marcin.maka@linux.intel.com>
+ */
+
+#ifndef __CAVS_LP_CTX_H__
+#define __CAVS_LP_CTX_H__
+
+#include <xtensa/xtruntime-frames.h>
+
+STRUCT_BEGIN
```

Comment:
Do we really want to use xtensa macros in platform/cavs folder?

Reply:
This is platform specific code. Would be nice to have all dependencies on the architecture abstracted, I agree. But atm our platforms use only xtensa architecture, so that kind of flexibility/abstraction is not really needed imo.